cmd/kubernetes/kubernetes_list_version.go

```diff
@@ -48,8 +50,18 @@ If you wish to use a custom format, the available fields are:
 
 			ow.StartLine()
 
-			ow.AppendDataWithLabel("version", version.Version, "Version")
-			ow.AppendDataWithLabel("type", version.Type, "Type")
+			if version.ClusterType == "" {
+				if strings.Contains(version.Label, "k3s") {
+					version.ClusterType = "k3s"
+				} else {
+					version.ClusterType = "talos"
+				}
+			}
```

Comment:
@satakshigarg could we investigate why the `cluster_type` is missing in the body of the response so I had to use this workaround?